From: Antonio Larrosa <alarrosa@suse.com>
Subject: Removes requirement for kf5-config (kdelibs4support)

With this, we don't need to include kdelibs4support and all its dependencies on SLE

Index: subversion-1.14.0/build/ac-macros/kwallet.m4
===================================================================
--- subversion-1.14.0.orig/build/ac-macros/kwallet.m4
+++ subversion-1.14.0/build/ac-macros/kwallet.m4
@@ -59,30 +59,13 @@ AC_DEFUN(SVN_LIB_KWALLET,
                 kde_lib_names="-lkdeui -lkdecore -lQtGui -lQtDBus -lQtCore"
               fi
               if test -n "$qt_pkg_config_names"; then
-                if test "$svn_lib_kwallet" != "yes"; then
-                  AC_MSG_CHECKING([for $kde_config_name])
-                  KDE_CONFIG="$svn_lib_kwallet/bin/$kde_config_name"
-                  if test -f "$KDE_CONFIG" && test -x "$KDE_CONFIG"; then
-                    AC_MSG_RESULT([yes])
+                  if echo "$svn_lib_kwallet" | $EGREP ":" > /dev/null; then
+                    kde_incdir=["`echo "$svn_lib_kwallet" | $SED -e "s/:.*//"`"]
+                    kde_libdir=["`echo "$svn_lib_kwallet" | $SED -e "s/.*://"`"]
                   else
-                    if echo "$svn_lib_kwallet" | $EGREP ":" > /dev/null; then
-                      AC_MSG_RESULT([unneeded])
-                      KDE_CONFIG="unneeded"
-                      kde_incdir=["`echo "$svn_lib_kwallet" | $SED -e "s/:.*//"`"]
-                      kde_libdir=["`echo "$svn_lib_kwallet" | $SED -e "s/.*://"`"]
-                    else
-                      AC_MSG_RESULT([no])
-                      KDE_CONFIG=""
-                    fi
-                  fi
-                else
-                  AC_PATH_PROG(KDE_CONFIG, $kde_config_name)
-                  if test -n "$KDE_CONFIG"; then
-                    kde_incdir="`$KDE_CONFIG --install include`"
-                    kde_libdir="`$KDE_CONFIG --install lib`"
+                    kde_incdir="${includedir}"
+                    kde_libdir="${libdir}"
                   fi
-                fi
-                if test -n "$KDE_CONFIG"; then
                   old_CXXFLAGS="$CXXFLAGS"
                   old_LDFLAGS="$LDFLAGS"
                   old_LIBS="$LIBS"
@@ -122,9 +105,6 @@ int main()
                     AC_MSG_RESULT([no])
                     AC_MSG_ERROR([cannot find KWallet])
                   fi
-                else
-                  AC_MSG_ERROR([cannot find $kde_config_name])
-                fi
               else
                 AC_MSG_RESULT([no])
                 AC_MSG_ERROR([cannot find Qt])
